app/models/comment.rb

```diff
@@ -1,3 +1,7 @@
 class Comment < ActiveRecord::Base
   belongs_to :commentable, :polymorphic => true
+
+  scope :recent, ->{ order('created_at DESC') }
```

Comment:
This scope confuses me. It is named `recent`, but it does not limit the comments based on time. Perhaps name it `by_creation_date` or something like that.

Reply:
Thinking about it more, you could even make this a default scope, because we would rarely want the unsorted.

Reply:
👍 sounds good to mee